package.json

```diff
@@ -150,6 +151,8 @@
     "react-markdown": "^2.5.0",
     "react-oembed-container": "^0.3.0",
     "react-overlays": "^0.8.3",
+    "react-primatives": "^0.0.5",
```

Comment:
Probably should remove this...

Reply:
roger that

Reply:
Woah, what even is this package that’s spelled ‘primitives’ instead of ‘primitives’? https://github.com/DavidWells/react-dom-primitives